src/Stratis.Bitcoin/P2P/PeerConnectorConnect.cs

```diff
@@ -18,20 +18,21 @@ public sealed class PeerConnectorConnectNode : PeerConnector
         /// <summary>Constructor for dependency injection.</summary>
         public PeerConnectorConnectNode(
             IAsyncLoopFactory asyncLoopFactory,
+            IDateTimeProvider dateTimeProvider,
             ILoggerFactory loggerFactory,
             Network network,
             INetworkPeerFactory networkPeerFactory,
             INodeLifetime nodeLifetime,
             NodeSettings nodeSettings,
             IPeerAddressManager peerAddressManager)
             :
```

Comment:
does the colon deserve its own line?

Reply:
OK I've moved it next to the constructor line.